doc/sphinx/mgmt_preview_quickstart.rst

```diff
@@ -0,0 +1,151 @@
+4. Click "Certificates & Secrets"
+5. Click "+ New client secret"
+6. Type description and click "Add"
+7. Copy and store the key value. You won’t be able to retrieve it after
+   you leave this page.
+
+Get Tenant ID
+^^^^^^^^^^^^^
+
+1. Login into your azure account
+2. Select azure active directory in the left sidebar
+3. Click "App Registrations" in the left sidebar
+4. Select the application which you have created
+5. Copy "Directory (tenant) ID"
+
+Setting Environment Variables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After you obtained the values, you need to set the following values as
+your environment variables
+
+-  ``AZURE_CLIENT_ID``
+-  ``AZURE_CLIENT_SECRET``
+-  ``AZURE_TENANT_ID``
+-  ``AZURE_SUBSCRIPTION_ID``
+
+To set the following environment variables on your development system:
+
+::
+
+    export AZURE_CLIENT_ID="__CLIENT_ID__"
+    export AZURE_CLIENT_SECRET="__CLIENT_SECRET__"
+    export AZURE_TENANT_ID="__TENANT_ID__"
+    export AZURE_SUBSCRIPTION_ID="__SUBSCRIPTION_ID__"
+
+Authentication and Creating REST Client
+---------------------------------------
+
+Now that the environment is setup, all you need to do is to create an
+authenticated client. Our default option is to use
+**DefaultAzureCredentials** and in this guide we have picked
+**Resource** as our target service. To authenticate to Azure and create
+a REST client, simply do the following:
+
+::
+
+    import azure.mgmt.resource
+    import azure.mgmt.network
+    import azure.mgmt.compute
+    from azure.identity import DefaultAzureCredential;
+    ...
+    subscription_id = os.environ.get("AZURE_SUBSCRIPTION_ID")
+    credentials = DefaultAzureCredential()
+    resource_client = azure.mgmt.resource.ResourceManagementClient(credential=credentials, subscription_id=subscription_id)
+
+More information and different authentication approaches using Azure Identity can be found at
+`here <https://docs.microsoft.com/en-us/python/api/overview/azure/identity-readme?view=azure-python>`__
+
+Managing Resources
+------------------
+
+Now that we are authenticated, we can use our REST client to make API
+calls. Let's create a resource group and demostrate REST client's usage
+
+**Create a resource group**
+
+::
+
+    location = "mylocation"
+    group_name = "my_resource_group_name"
+    group = resource_client.resource_groups.create_or_update(
+        group_name,
+        {'location': location}
+    )
+
+**Update a resource group**
+
+::
+
+    group_name = "my_resource_group_name"
+    group.tags = {
+        "environment":"test",
+        "department":"tech"
+    }
+    updated_group = resource_client.resource_groups.create_or_update(group_name, group)
+
+**List all resource groups**
+
+::
+
+    # 
+    group_list = self.resource_client.resource_groups.list()
+    for g in group_list:
+        print_resource_group(g)
+
+**Delete a resource group**
+
+::
+
+    delete_async_op = resource_client.resource_groups.begin_delete(group_name)
+    delete_async_op.wait()
```

Comment:
Is this document going to appear as a README for mgmt plane? If yes might be worth adding links to:
- more samples
- troubleshooting
- contributing section

Reply:
yes, we are working on the samples part, when it's setup i'll add link to the samples here.
For troubleshooting and contributing sections, are there existing places we can point to?

Reply:
For both of these sections check what we have in the main readme. Troubleshooting could encourage posting an issue, link to ref docs, stack overflow. Contributing is just a copy paste: 

### Contributing
For details on contributing to this repository, see the [contributing guide](https://github.com/Azure/azure-sdk-for-python/blob/master/CONTRIBUTING.md).

This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.

When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repositories using our CLA.

This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact opencode@microsoft.com with any additional questions or comments.

Reply:
I think we could add appropriate links to reference for resource group, that would teach people how to find appropriate section in python reference.
However for now we could add links to latest versions and later remember to change it to bx version.

Reply:
thanks Zim, I will add links once track 2 api references are updated